# Upstream status: in linux-mips tree; someone needs to figure out how to
# integrate this into the Linux tree...

From: Maciej W. Rozycki <macro@linux-mips.org>

--- a/drivers/scsi/NCR53C9x.h	2006-03-05 20:35:04.000000000 +0100
+++ b/drivers/scsi/NCR53C9x.h	2006-03-05 19:51:16.000000000 +0100
@@ -145,12 +145,7 @@
 
 #ifndef MULTIPLE_PAD_SIZES
 
-#ifdef CONFIG_CPU_HAS_WB
-#include <asm/wbflush.h>
-#define esp_write(__reg, __val) do{(__reg) = (__val); wbflush();} while(0)
-#else
-#define esp_write(__reg, __val) ((__reg) = (__val))
-#endif
+#define esp_write(__reg, __val) do{(__reg) = (__val); iob();} while(0)
 #define esp_read(__reg) (__reg)
 
 struct ESP_regs {
